src/overlays/actors/ovl_En_Tg/z_en_tg.c

```diff
@@ -15,23 +15,12 @@ void EnTg_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnTg_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnTg_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_06005040;
-extern UNK_TYPE D_0600AE40;
+void EnTg_SpinIfNotTalking(EnTg* this, GlobalContext* globalCtx);
 
-/*
-const ActorInit En_Tg_InitVars = {
-    ACTOR_EN_TG,
-    ACTORCAT_NPC,
-    FLAGS,
-    OBJECT_MU,
-    sizeof(EnTg),
-    (ActorFunc)EnTg_Init,
-    (ActorFunc)EnTg_Destroy,
-    (ActorFunc)EnTg_Update,
-    (ActorFunc)EnTg_Draw,
-};
+extern AnimationHeader D_06005040;
+extern FlexSkeletonHeader D_0600AE40;
```

Comment:
```suggestion
```

these are part of object_mu which is extracted by zapd, so you can eliminate these extern references. To do this properly, you need to add

`#include objects/object_mu/object_mu.h`

below `#include "z_en_tg.h"`

Then, you simply replace `D_06005040` with the proper symbol defined in object_mu.xml 
I'll add another review comment below to show how this should look.

Reply:
You can then remove these two `D_*` symbols from `undefined_syms.txt` (file at the root of the repository)